src/main/java/org/commcare/formplayer/application/MenuController.java

```diff
@@ -163,8 +166,14 @@ public EntityDetailListResponse getDetails(@RequestBody SessionNavigationBean se
     @UserRestore
     @AppInstall
     public BaseResponseBean navigateSessionWithAuth(@RequestBody SessionNavigationBean sessionNavigationBean,
-            @CookieValue(Constants.POSTGRES_DJANGO_SESSION_ID) String authToken,
-            HttpServletRequest request) throws Exception {
+                        @CookieValue(Constants.POSTGRES_DJANGO_SESSION_ID) String authToken,
+                        @CookieValue(value = FORMPLAYER_SESSION_LANGUAGE, required = false) String sessionPrefLang,
```

Comment:
nit: Constants is already imported:
```suggestion
                        @CookieValue(Constants.FORMPLAYER_SESSION_LANGUAGE, required = false) String sessionPrefLang,
```